src/components/SurveyList/index.tsx

```diff
@@ -0,0 +1,48 @@
+/* eslint-disable @next/next/no-img-element */
```

Comment:
I'm just curious, doesn't it work using NextJS [Image](https://nextjs.org/docs/messages/no-img-element) element instead of an `<img />` tag? 🤔

Reply:
To use the NextJS image with an image from a remote URL, we need to add all the remotes domain to the NextJS configuration. But survey image domain can be changed depending on the API response so I prefer using plain img tag instead
Ref: https://nextjs.org/docs/basic-features/image-optimization#domains

Reply:
@malparty To use the NextJS image with an image from a remote URL, we need to add all the remotes domain to the NextJS configuration. But survey image domain can be changed depending on the API response so I prefer using plain img tag instead
Ref: https://nextjs.org/docs/basic-features/image-optimization#domains

Reply:
Thanks for your clear explanation :) 

![image](https://user-images.githubusercontent.com/77609814/174237139-7dd41bd1-43c5-4bbd-8339-59cba74ec4d5.png)